src/db.c

```diff
@@ -2004,6 +2027,31 @@ db_files_get_count_bymatch(char *path)
 #undef Q_TMPL
 }
 
+int
+db_file_get_seekpos(uint32_t id)
```

Comment:
db_file_get_seekpos -> just use db_file_fetch_byid (or save the seek in the queue?)